tests/resources/generator/hodgkin_huxley_squid_axon_model_1952/model.algebraic.c

```diff
@@ -56,7 +56,7 @@ void deleteArray(double *array)
     free(array);
 }
 
-void initialiseStatesAndConstants(double *states, double *variables)
+void initialiseVariables(double voi, double *states, double *variables, ExternalVariable externalVariable)
```

Comment:
The name of the file is mileading. `algebraic` relates to the external variable, i.e. we originally have an algebraic variable (`variables[5]`) which we, here, decided to flag as an external variable.

Regarding the `voi`, it's here because the model is indeed a differential model, independent of whether we have flagged some variables as being external.

Reply:
okay, just checking that that was the case.